### DISCUSSION TITLE
How to quickly clip and paste files with pictures to outline

### DISCUSSION THREAD

**Discussion by WarlockFish** [outline/outline#5069] [General]
Sometimes, the articles are very good, and I want to import them into the outline. I can use the "[Web Clipper](https://github.com/webclipper/web-clipper)"  to quickly save text    to markdown file  and import the outline, but images cannot be quickly imported. Is there any way to quickly import text and images

**Comment by tommoor** [comment]
You can copy and paste content from any webpage into outline, it should come across well in most circumstances

**Reply by WarlockFish** [reply]
Yes, it's very good most of the time 。
I have such a usage scenario: there is an article with a lot of pictures, and I need to save it to the outline. If I copy and paste it directly, the link of the picture is the source URL.  Sometimes, the source page may fail or the page may not be maintained ,Is there any way to save it directly to the local S3 storage?

**Comment by leonisir** [comment]
There is also such a need

**Comment by bestbugwriter** [comment]
There is also such a need

**Comment by baditaflorin** [comment]
This seems to be a request for converting images to local storage
